File: Java/Java - Programmieren für Einsteiger/uebung2b.java
/*
2.  Betrachten Sie nochmals das erste Programm dieses Lehrbuchs (Kapitel III.1.), das den Text "Herzlich Willkommen zum Java-Kurs!" ausgibt.
    Schreiben Sie ein Programm, das genau die gleiche Funktion erfüllt.
    Dabei soll jedoch ein Array gestaltet und dabei jedes einzelne Wort in einem Feld gespeichert werden.
    Geben Sie daraufhin die einzelnen Felder nacheinander aus.

Bonacina, Michael. Java Programmieren: für Einsteiger: Der leichte Weg zum Java-Experten (2. Auflage: komplett neu verfasst) (Einfach Programmieren lernen 1) (German Edition) (S.58-60). Kindle-Version.
*/

public class uebung2b
{
    public static void main(final String[] args)
    {
        String[] x = {"Herzlich", "Willkommen", "zum", "Java-Kurs!"};

        System.out.print(x[0] + " " + x[1] + " " + x[2] + " " + x[3]);
    }
}

File: C++/C++ - Programmieren für Einsteiger/uebung2b.cpp
/*
2.  Erstellen Sie ein Programm mit den Integer-Variablen a und b und weisen Sie diesen beliebige Werte zu.
    Addieren Sie a und b und weisen Sie diesen Wert der Variablen a zu.
    Multiplizieren Sie den neuen Wert für a mit b und weisen Sie diesen erneut a zu und geben Sie den Wert von a auf dem Bildschirm aus.
    Erstellen Sie zwei verschiedene Lösungswege für die mathematischen Operationen.

Bonacina, Michael. C++ Programmieren: für Einsteiger: Der leichte Weg zum C++-Experten (Einfach Programmieren lernen 3) (German Edition) (S.52). Kindle-Version.
*/

#include <iostream>

using namespace std;

int main()
{
    int x = 2;
    int y = 3;
    int tmp = 0;

    tmp = x + y;
    tmp = tmp * y;

    cout << "Ergebnis (v1): " << tmp << endl;

    x += y;
    x *= y;

    cout << "Ergebnis (v2): " << x << endl;

    return 0;
}